### DISCUSSION TITLE
where's java agent in apache-skywalking-apm-8.8.1.tar.gz ?

### DISCUSSION THREAD

**Discussion by 1091643978** [apache/skywalking#7957] [General]
I download the apache-skywalking-apm-8.8.1.tar.gz, and cannot find the agent folder according to doc's description.
```
~/tools/apache-skywalking-apm-bin|⇒  ls
bin  config  config-examples  LICENSE  licenses  NOTICE  oap-libs  README.txt  tools  webapp
```

according https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/readme/
```
Setup java agent
Agent is available for JDK 8 - 17.
Find agent folder in SkyWalking release package
Set agent.service_name in config/agent.config. Could be any String in English.
Set collector.backend_service in config/agent.config. Default point to 127.0.0.1:11800, only works for local backend.
Add -javaagent:/path/to/skywalking-package/agent/skywalking-agent.jar to JVM argument. And make sure to add it before the -jar argument.
The agent release dist is included in Apache official release. New agent package looks like this.
```
official release url:http://skywalking.apache.org/downloads/

 Agent:
 SkyWalking Nginx LUA
 SkyWalking Kong
 SkyWalking Python
 SkyWalking NodeJS
 SkyWalking Client JavaScript
 SkyWalking Satellite

and Docker Images for convenience, there's a java agent docker image. 

where's the java agent?

"The agent release dist is included in Apache official release." where is it?

**Comment by wu-sheng** [comment]
https://github.com/apache/skywalking/discussions/7548

Read this, and 8.8.1 release doc has also provided the information.
